drivers/bluetooth/hci/Kconfig

```diff
@@ -30,6 +30,17 @@ config BT_H5
 	  Bluetooth three-wire (H:5) UART driver. Implementation of HCI
 	  Three-Wire UART Transport Layer.
 
+config BT_NRF_OPEN_AMP
+	bool "Open AMP HCI"
+	select IPM
+	select IPM_NRFX
+	select NRFX_IPC
+	select IPM_MSG_CH_1_ENABLE
```

Comment:
I would like to configure channel direction here. However, this syntax does not work

```
select IPM_MSG_CH_0_TX
select IPM_MSG_CH_1_RX
```

since IPM_MSG_CH_0_TX is a ```choice``` item. How can I work around this?

Reply:
Could you do what we did with choosing a libc implementation?

Add a layer of indirection by having the user choose something that affected the default choice.

https://github.com/zephyrproject-rtos/zephyr/blob/master/lib/libc/Kconfig#L17

Reply:
Unfortunately, the choice option is defined within a Kconfig template file (the template file depends on channel number), so it would be messy to have it there (if possible at all)

https://github.com/zephyrproject-rtos/zephyr/blob/9523043d97ce6ed895a02185378971c67e342846/drivers/ipm/Kconfig.nrfx_ipc_channel

Reply:
It is also impossible to give a parameterized name to the choice option:
```choice IPM_MSG_CH_$(nrfx_ipc_num)_DIR```
Kconfig python parser scripts cannot handle that.

With name, I could define the default in the Kconfig board file:
```
choice IPM_MSG_CH_0_DIR
	default IPM_MSG_CH_0_TX if BT_NRF_OPEN_AMP
endchoice
```